samples/openthread/ncp/src/user_vendor_hook.cpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * @file
+ *   This file shows how to implement the NCP vendor hook.
+ */
+#if OPENTHREAD_ENABLE_NCP_VENDOR_HOOK
+#include "nrf_802154_radio_wrapper.h"
+#include <logging/log.h>
+#include "ncp_base.hpp"
+#include "ncp_uart.hpp"
+#include "common/new.hpp"
+#include <assert.h>
+
+#define VENDOR_SPINEL_PROP_VENDOR_NAME 0x3c11
```

Comment:
isin't there a VENDOR_START or something so that if  the values change it gets updated here?

Reply:
Right, done